.gitignore

```diff
@@ -31,6 +31,9 @@ coverage.txt
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# vim swap files
+*.swp
```

Comment:
What do you think about `*.sw?` since vim generates .swp, .swo, .swn, etc. for it's various swap files